src/Scraper/Application/Configurator.php

```diff
@@ -75,19 +75,26 @@ private function getCrawler($scrapedData)
      */
     private function findConfigByScrapedData($scrapedData, $crawler)
     {
-        $result = [];
+        $result         = [];
+        $variant        = [];
+        $allFieldsFound = true;
+
         foreach ($scrapedData['data'] as $field => $value) {
             try {
                 $result[$field] = $this->xpathBuilder->find(
                     $crawler->getNode(0),
                     $value
                 );
+                $variant[]      = $field . $result[$field];
```

Comment:
Wouldn't you think that if this was a collection of objects it would be easier to read?

Reply:
Some example?

Reply:
Some example?